doc/api_assets/api.js

```diff
@@ -136,6 +136,36 @@
     updateHashes();
   }
 
+  function setupCopyButton() {
+    const buttons = document.querySelectorAll('.copy-button');
+    buttons.forEach((button) => {
+      button.addEventListener('click', (el) => {
```

Comment:
The event handler may be able to extract.

Reply:
Hi @btea thank you for your feedback, I am just not too sure if it is worth the effort of function extraction.
* The code is temp and won't live in the future doc tool
* the function is relatively short and straightforward

Reply:
Yeah, what you said makes sense.  I made this point because each page of the nodejs official website has a lot of content, and some have many code blocks, so there may be many bindings of the same event, so I think it is worth extracting. 

Anyway, I think this feature looks great 👍